feat: Added support for iOS home screen icons

paschal533 opened the pull request:
## Description

Added multiple apple-touch-icon sizes to support iOS devices when the app is saved to the home screen. 

- Included apple-touch-icons of various sizes (72x72, 114x114, 120x120, 144x144, 152x152, 167x167, 180x180).
- Updated meta tags for web app capability and status bar style.
- Ensures proper display when the app is saved on iOS home screens.
 
## Related Issues

fixed #238

betamax commented:
Hi Paschal, thanks. Couple of comments:

1. I did a bit of reading and I think for optimal support we don't need to have sizes for each case, so can we just have the sizes: 

    - 180px × 180px
    - 120px × 120px
    - 167px × 167px
    - 152px × 152px

2. Can you please remove the comments? I don't think they're necessary.

@felixfelixfelix do you think the icon that was used here is acceptable?

paschal533 commented:
> Hi Paschal, thanks. Couple of comments:
> 
> 1. I did a bit of reading and I think for optimal support we don't need to have sizes for each case, so can we just have the sizes:
>    
>    * 180px × 180px
>    * 120px × 120px
>    * 167px × 167px
>    * 152px × 152px
> 2. Can you please remove the comments? I don't think they're necessary.
> 
> @felixfelixfelix do you think the icon that was used here is acceptable?

Hi Max,

Thanks for the feedback! 😊

- I’ve updated the code to include only the recommended apple-touch-icon sizes: 120x120, 152x152, 167x167, and 180x180.
- I’ve also removed the unnecessary comments for a cleaner implementation.
Let me know if there’s anything else you’d like me to adjust!

betamax commented:
Thanks, I am just waiting to hear back from Felix about the icon then I think it's good to merge

paschal533 commented:
Hi @betamax ,
I’ve updated the images as requested.

Btw, I’m still experiencing some issues while trying to sign up on the website through Docker. Let me know when you’d be free to help me troubleshoot the problems I’m encountering.

Thanks

betamax commented:
Thanks, merged!

Can you post your questions regarding Docker in the other thread where you were originally asking about Docker? Including what the problems you are encountering are. I can respond then there.
